test/stablehlo/test_stablehlo_custom_call.py

```diff
@@ -115,6 +116,22 @@ def forward(self, x):
     # self.assertTrue("api_version = 1" in shlo_text)
 
 
+  def test_place_to_host_device(self):
+    dev = xm.xla_device()
+    a = torch.ones(10, device=dev)
+    b = place_to_host(a)
+    shlo_text = xm.get_stablehlo([b])
+    self.assertTrue("has_side_effect = true" in shlo_text)
+    self.assertTrue("mhlo.frontend_attributes = {_xla_buffer_placement = \"pinned_host\"}}" in shlo_text)
+
+    a = torch.ones(10, device=dev)
+    b = place_to_device(a)
+    shlo_text = xm.get_stablehlo([b])
+    self.assertTrue("has_side_effect = true" in shlo_text)
+    self.assertTrue("mhlo.frontend_attributes = {_xla_buffer_placement = \"device\"}}" in shlo_text)
```

Comment:
In the design doc IIRC it said "hbm". Do you know if Jax produces "device" in practice?

Reply:
Never mind, saw your example.

Reply:
Yes, see the commit message